taichi/codegen/llvm/codegen_llvm.cpp

```diff
@@ -688,6 +688,11 @@ llvm::Type *TaskCodeGenLLVM::llvm_type(DataType dt) {
     return llvm::Type::getDoubleTy(*llvm_context);
   } else if (dt->is_primitive(PrimitiveTypeID::f16)) {
     return llvm::Type::getHalfTy(*llvm_context);
+  } else if (dt->is<TensorType>()) {
```

Comment:
1. Shall we assert `real_matrix=True` here?
2. It seems that this function is exactly the same as `llvm::Type *TaichiLLVMContext::get_data_type(DataType dt)`. Can we merge them as well (in another PR maybe).

Reply:
Sounds good to me